Assets/Python/StoredData.py

```diff
@@ -150,6 +153,24 @@ def getTimedConquests(self):
 	def resetTimedConquests(self):
 		self.scriptDict['lTimedConquests'] = []
 		
+	def setPlayerEnabled(self, iCiv, bNewValue):
+		self.scriptDict['lPlayerEnabled'][con.lSecondaryCivs.index(iCiv)] = bNewValue
+		
+	def getPlayerEnabled(self, iCiv):

```

Comment:
Just a small comment, by convention getter methods returning a boolean are usually called isPlayerEnabled() etc., same for the other ones here.